### ISSUE TITLE
Method visualization is broken

### ISSUE THREAD

**Issue by sdanyliv** [agileobjects/ReadableExpressions#61]
Try to visualize any of these methods.
```cs
var methods = typeof(Queryable).GetRuntimeMethods()
    .Where(e => e.Name == "SelectMany").ToArray();
```
Invalid output:
```cs
public static IQueryable Queryable.SelectMany
(
    IQueryable source,
    Expression<FUNC>> selector
)
```
Same issue with visualizing types.

Version 3.0.1

**Comment by SteveWilkes** [comment]
Thanks for letting me know - this is fixed in the latest v3.1 release branch, release to follow.

Cheers!

Steve

**Comment by SteveWilkes** [comment]
This is fixed as of v3.1, which is now in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1232914.ReadableExpressionsVisualizers).

Thanks again for letting me know!

Steve
